test/mocha/mig-login.js

```diff
@@ -0,0 +1,27 @@
+/* global describe before after it expect Event */
+import closest from '../../app/utils/closest'
+import {dispatch} from '../../app/utils/events'
+import app from '../..' // eslint-disable-line no-unused-vars
```

Comment:
That's how I make sure the app is loaded in the test runner. It gets compiled with Browserify in the same bundle as the tests. Alternative would be to bundle separately, include it via Karma, and run tests against that black box, but that's even more un-unit-test-like (and probably tricky to keep it fast with watchify). :-/